mobile/services/Tor/index.ts

```diff
@@ -0,0 +1,87 @@
+import Tor from 'react-native-tor';
+
+class TorClient {
+  baseUrl: string;
+  daemon: ReturnType<typeof Tor>;
+
+  constructor() {
+    this.baseUrl = 'http://robosats6tkf3eva7x2voqso3a5wcorsnw34jveyxfqi2fu7oyheasid.onion';
+    this.daemon = Tor({
+      stopDaemonOnBackground: false,
+      numberConcurrentRequests: 0,
```

Comment:
@Reckless-Satoshi According to https://github.com/Sifir-io/react-native-tor/blob/ed9dc6f1c474c3aa1ae7512f7437343159a46480/src/index.tsx#L349 we are safe to set this to 0 (we are using v8) and it should resolve the error you got. I tested loading all pages and didn't get any error.